src/relay/op/tensor/transform.cc

```diff
@@ -254,6 +254,12 @@ bool ConcatenateRel(const Array<Type>& types,
     if (e_dtype != dtype) {
       throw relay::Error("relay.concatenate requires all tensors have the same dtype");
     }
+    for (int j = 0; j < first->shape.size(); ++j) {
+      if (j == param->axis) continue;
```

Comment:
`static_cast<size_t>(param->axis)` if j is `size_t`

Reply:
Fixed.